src/main/webapp/assets/modules/common/securityService.js

```diff
@@ -17,8 +17,14 @@ techlooper.factory("securityService", function (apiService, $rootScope, $q, util
 
       apiService
         .login({us: $.cookie("us"), pwd: $.cookie("pwd")})
-        .success(function (data, status, headers, config) {$rootScope.$emit("$loginSuccess");});
-
+        .success(function (data, status, headers, config) {
```

Comment:
'config' is defined but never used.<br>'headers' is defined but never used.<br>'status' is defined but never used.<br>'data' is defined but never used.